### PR TITLE
[SEMI-MODULAR] Re-adds the stock market for cargo

### PR DESCRIPTION
**Pull request by DuffCreeper**
## About The Pull Request

So uh, this was stupidly easy to bring back. Anyway this basically adds the stock market for cargo to fuck with and either earn money or lose it

## Why It's Good For The Game

So this gives cargo a new (old) gimmick to fuck with and another way to earn money other than people throwing stuff at them.
Nothing is changed about this and should perform exactly the same as the day TG removed it.

You will find the circuitboard can either be printed or be found in the QMs locker roundstart. Reason being so cargo techs dont just yeet all the money into stocks round start, having it in the QMs locker gives them some form of control.

But uh yeah, not sure what else to say about this. Anyone who used it before will know exactly what it is

## Changelog
:cl:
add: Re-added the stock market to cargo. You can find the board in the QMs locker round start or print it. Remember, sell high buy low
/:cl:

### CODE REVIEW

**Comment by RatFromTheJungle** [comment]
cacogo

File: modular_skyrat/modules/stock_market/code/articles.dm [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+/proc/ucfirst(var/S)
+	return "[uppertext(ascii2text(text2ascii(S, 1)))][copytext(S, 2)]"
+
+/proc/ucfirsts(var/S)
+	var/list/L = splittext(S, " ")
+	var/list/M = list()
+	for (var/P in L)
+		M += ucfirst(P)
+	return jointext(M, " ")
```
Comment (tf-4) [inline modular_skyrat/modules/stock_market/code/articles.dm]: Look at all these *single letter variables*

File: modular_skyrat/modules/stock_market/code/articles.dm [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	var/opinion = 0
+	var/ticks = 0
+	var/datum/stock/about = null
+	var/outlet = ""
+	var/static/list/outlets = list()
+	var/static/list/default_tokens = list( \
+		"buy" = list("buy!", "buy, buy, buy!", "get in now!", "ride the share value to the stars!"), \
+		"company" = list("company", "corporation", "conglomerate", "enterprise", "venture"), \
+		"complete" = list("complete", "total", "absolute", "incredible"), \
+		"country" = list("Space", "Argentina", "Hungary", "United States of America", "United Space", "Space Federation", "Nanotrasen", "The Wizard Federation", "United Kingdom", "Poland", "The Syndicate", "Australia", "Serbia", "The European Union", "The Illuminati", "The New World Order", "Eurasian Union", "Asian Union", "United Arab Emirates", "Arabian League", "United States of Africa", "Mars Federation", "Allied Colonies of Jupiter", "Saturn's Ring", "Fringe Republic of Formerly Planet Pluto"), \
```
Comment (tf-4) [inline modular_skyrat/modules/stock_market/code/articles.dm]: Better to replace stuff like Australia or the UAE with stuff from our backgrounds section

File: modular_skyrat/modules/stock_market/code/articles.dm (hunk 1 above)
Comment (DuffCreeper) [reply modular_skyrat/modules/stock_market/code/articles.dm]: The god and legend prodirus went out of his way to uncancer this all to actual meanings

File: modular_skyrat/modules/stock_market/code/articles.dm (hunk 2 above)
Comment (DuffCreeper) [reply modular_skyrat/modules/stock_market/code/articles.dm]: Should be good I think

**Comment by KasparoVy** [comment]
Thanks much for the lore related changes! Tying it into backgrounds is appreciated.

**Comment by DuffCreeper** [comment]
Suppose at this point if peeps are happy for this to be merged it can be merged. Nothing else I wanna touch at this point

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**